packages/app/src/components/catalog/EntityPage/Content/Overview.tsx

```diff
@@ -182,5 +186,11 @@ export const overviewContent = (
         </Grid>
       </EntitySwitch.Case>
     </EntitySwitch>
+
+    <EntitySwitch>
+      <EntitySwitch.Case if={isJenkinsAvailable}>
+        <EntityLatestJenkinsRunCard branch="" />
```

Comment:
Does the `branch` prop need to be specified?

Reply:
Yeah, it's a required prop. I left it blank to specify all branches.